test/jdk/java/awt/TrayIcon/ActionCommand/ActionCommand.java

```diff
@@ -140,7 +158,7 @@ void doTest() throws Exception {
             }
         });
 
-        robot.mouseMove(0, 0);
+        robot.mouseMove(100, 0);
```

Comment:
I moved it away from the top left corner, because it can trigger a `hot corner` in Gnome(X11), so the following tests may fail.